### PR TITLE
Update xkcd tutorial for 0.32

### PR DESCRIPTION
**Pull request by jasongrout**
See also #4377

### CODE REVIEW

File: docs/source/developer/xkcd_extension_tutorial.rst [hunk 1]
```diff
@@ -259,6 +259,13 @@ to get a reference to the command palette interface.
       ICommandPalette
     } from '@jupyterlab/apputils';
 
+We will also need to install this dependency. Run the following command in the
+repository root folder.
```
Comment (ian-r-rose) [inline docs/source/developer/xkcd_extension_tutorial.rst]: For this and the other additions of this nature: I think it reads more smoothly to have it end in a ":".

Also, I see there is some mix in the voice of this article. There is some first person plural ("we"), but much more if it is second-person singular. I think that the latter conforms more closely to our documentation [guidelines](https://github.com/jupyterlab/jupyterlab/blob/master/docs/source/developer/documentation.rst#writing-style).

Comment (jzf2101) [reply docs/source/developer/xkcd_extension_tutorial.rst]: I agree

File: docs/source/developer/xkcd_extension_tutorial.rst [hunk 2]
```diff
@@ -318,6 +325,13 @@ top of the file.
       Widget
     } from '@phosphor/widgets';
 
+We will also need to install this dependency. Run the following.
```
Comment (jzf2101) [inline docs/source/developer/xkcd_extension_tutorial.rst]: @ian-r-rose 's comment applies here

File: docs/source/developer/xkcd_extension_tutorial.rst [hunk 3]
```diff
@@ -536,6 +550,13 @@ Add the following additional import to the top of the file.
       Message
     } from '@phosphor/messaging';
 
+We will also need to install this dependency. Run the following.
```
Comment (jzf2101) [inline docs/source/developer/xkcd_extension_tutorial.rst]: Here also

File: docs/source/developer/xkcd_extension_tutorial.rst [hunk 4]
```diff
@@ -696,6 +717,13 @@ entire list of import statements looks like the following:
 
     import '../style/index.css';
 
+
+We will also need to install the new dependency. Run the following.
```
Comment (jzf2101) [inline docs/source/developer/xkcd_extension_tutorial.rst]: Also here

File: docs/source/developer/xkcd_extension_tutorial.rst (hunk 1 above)
Comment (jasongrout) [reply docs/source/developer/xkcd_extension_tutorial.rst]: "We" - Sorry, still have the habit from math papers :).

Comment (jasongrout) [reply docs/source/developer/xkcd_extension_tutorial.rst]: > For this and the other additions of this nature: I think it reads more smoothly to have it end in a ":".

The commands in the first part all don't have the colon. There are a few in the last part of the article that do have a colon. We should probably clean that up on another PR.

**Comment by jasongrout** [comment]
Changes addressed - feel free to merge, or to edit if you like.

**Comment by ian-r-rose** [comment]
Test failures are to due issues with PyPI. Merging, thanks @jasongrout1

**Pull request merged**